client/state/themes/actions.js

```diff
@@ -133,32 +136,30 @@ export function receiveServerError( error ) {
  * Returns an action object to be used in signalling that a theme activation
  * has been triggered
  *
- * @param  {Object}  theme Theme received
- * @param  {Object}  site Site used for activation
+ * @param  {Number}  themeId Theme to be activated
```

Comment:
(There are a couple more instances of this JSDoc argument type in this commit that need fixing as well.)